defaults/main.yml

```diff
@@ -331,6 +331,9 @@ camac_caluma_languages: "de,fr"
 # Enable caluma historical API
 camac_caluma_enable_historical_api: true
 
+# Enable Manabi (WebDAV)
+manabi_enable: false
```

Comment:
I'd say we flip the default once the feature is ready for prime time, agreed?

Reply:
Yes, for now we'll leave it like that